### PR TITLE
Update Set-ThrottlingPolicyAssociation.md

### PR DESCRIPTION
**Pull request by v-smandalika**

### CODE REVIEW

File: exchange/exchange-ps/exchange/Set-ThrottlingPolicyAssociation.md [hunk 1]
```diff
@@ -14,9 +14,13 @@ ms.reviewer:
 ## SYNOPSIS
 This cmdlet is available only in on-premises Exchange.
 
-Use the Set-ThrottlingPolicyAssociation cmdlet to associate a throttling policy with a specific object. The object can be a user with a mailbox, a user without a mailbox, a contact or a computer account.
+Use the Set-ThrottlingPolicyAssociation cmdlet to associate a throttling policy with a specific object. The object can be a user with a mailbox, a user without a mailbox, a contact, or a computer account.
 
-For information about the parameter sets in the Syntax section below, see [Exchange cmdlet syntax](https://learn.microsoft.com/powershell/exchange/exchange-cmdlet-syntax).
+**Note**: Some parameters in the throttling policy (for example, MessageRateLimit) apply only to objects that have mailbox GUIDs (mailboxes or remote mailboxes) and don't apply to mail users.
+
+For remote mailbox, use the Set-RemoteMailbox -ExchangeGUID cmdlet, if ExchangeGUID isn't populated.
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/Set-ThrottlingPolicyAssociation.md:21]:
```suggestion
For remote mailboxes, use the Set-RemoteMailbox -ExchangeGUID cmdlet if ExchangeGUID isn't populated.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @v-smandalika, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by v-smandalika** [comment]
Hi @AshaIyengar21 ,

Apologies I missed assigning this PR to you earlier, when I had emailed you this PR for a validation of my self-review changes.

Thanks,
@v-smandalika

File: exchange/exchange-ps/exchange/Set-ThrottlingPolicyAssociation.md [hunk 2]
```diff
@@ -32,11 +34,11 @@ Set-ThrottlingPolicyAssociation [-Identity] <ThrottlingPolicyAssociationIdParame
 ## DESCRIPTION
 The Set-ThrottlingPolicyAssociation cmdlet defines quota limits for specific objects. For example, if you notice that a user or other object is using excessive bandwidth, you can associate that object with a throttling policy that's more restrictive.
 
-In data center deployments, the object referred to by the Identity and ThrottlingPolicy parameters must be in the same tenant.
+In data center deployments, the Identity parameter-referred object and the ThrottlingPolicy parameter-referred object must be in the same tenant.
```
Comment (AshaIyengar21) [inline exchange/exchange-ps/exchange/Set-ThrottlingPolicyAssociation.md:37]: Please retain the previous sentence as is..your proposed change, changes the meaning of the sentence itself and hence seem to be inaccurate

**Comment by v-smandalika** [comment]
Hi @claydetels19 ,

SME Bhalchandra Atre has approved the updates we have made in this PR; and, the internal-review comments have also been incorporated.

We are hereby assigning the PR to you for help in reviewing it and in assigning the Sign off label.


Thanks,
@v-smandalika

**Pull request merged**